contrib/test/integration/build/cri-o.yml

```diff
@@ -66,7 +66,7 @@
     dest: /etc/crio/crio.conf
     line: '"docker.io"'
     insertafter: 'registries = \['
-    regexp: 'docker\.io'
+    regexp: 'registries = \[ "docker\.io" \]'
     state: present
```

Comment:
mmmmm, no, I don't think that's right.  Probably also need to remove the ``insertafter: 'registries = \['`` argument.

Reply:
...reading docs...

Reply:
yeah, ``lineinfile`` is really messy to use in this case.   If the format of the file changes, breakage is likely, and keeping the regex in-sync is a PITA.  If there's no regex-match, ``insertafter`` will be taken to mean EOF...not what you want. 
 Better just do:

```
- name: blah
  copy:
    dest: /etc/crio/crio.conf
    content: |
         # File added by ansible from build/cri-o.yml
         ....file contents.....
```